Userland/Utilities/ps.cpp

```diff
@@ -20,14 +20,22 @@ static ErrorOr<String> determine_tty_pseudo_name()
         perror("fstat");
         return Error::from_errno(saved_errno);
     }
+
     int tty_device_major = major(tty_stat.st_rdev);
     int tty_device_minor = minor(tty_stat.st_rdev);
+
     if (tty_device_major == 201) {
         return String::formatted("pts:{}", tty_device_minor);
     }
+
     if (tty_device_major == 4) {
         return String::formatted("tty:{}", tty_device_minor);
     }
+
+    if (tty_device_major == 1 && tty_device_minor == 3) {
+        return String::formatted("tty:{}", tty_device_minor);
```

Comment:
When you already know the device minor, there is no need to call String::formatted, just hardcode the string